cmd/manager/aggregator.go

```diff
@@ -336,6 +337,31 @@ func createOrUpdateOneResult(crClient aggregatorCrClient, owner metav1.Object, l
 	return nil
 }
 
+func setTimestampAnnotations(crClient aggregatorCrClient, owner metav1.Object, annotations map[string]string) map[string]string {
+	// Get the scan timestamp
+	scan := &compv1alpha1.ComplianceScan{}
```

Comment:
From a network chatter perspective, we should only need to get this value once before we start creating or update each `ComplianceCheckResult`, right?

If we bury this in the `createOrUpdateOneResult` implementation I think we'll be fetching the scan start time for each result.

Reply:
that's a good idea, fixing it

Reply:
Aren't we still fetching the ComplianceScan with each ComplianceCheckResult?